.travis.yml

```diff
@@ -6,33 +7,66 @@ env:
     - secure: "UC5rEkKAKQHvpCojJAedrx1aVsplZ0xOq5xdFME3UxrKDOsihRIDBYV6YOnpDcWVxzI3iZAJOJG1rNqMZ/4pOnI/DQ5n2nCIbqZlhPh6xPbr//g88Y3dlQifPWbC6pqmh+WJCfde99lYNDI2XauD4AvRARI9MnRP9vGF0gyTbUMeA0ybLdXScawnhL48Gh7nj64R7e0F1SgF1iEXaU0LvWUW2UK9yIg/kqWfL0XKpgpvxcy0n+VGe9gbwiA70F3nd7iZ/aM7K1MRiLeppCbj3xlLwGE4mw6svF5uim6oYZoDp0si70Ux2yw7TNOYpn82dTBIq1KU59jGUyP/rH/nx0erPzCUJFak8YQTSrbhO08u/baRt5Qb0KWfqGJZcHzPknRC66PzI47zO5ol9qvnIo68xfuIrlJZRhTF7ewNWzmJP5MeJ85mgoQSTmZYyCgFTYDcPdoePerhjBepYngvT1EzS9BPTuUXhStH7lRgbriGVreOQ9bDvW5hTb/n9CKmI/XhQBOGy7T1HYAaIP1N+rlQw0lmp2uYIvZ53wGjqoKB+yLBpAAzXPBbK3sjDtuquou/jEqvyJnL9/VEbohPfIK3AlTTC0zIDi5QZq/h4zqMIKOybd3jmvT4P+Pvy8YGolXtlV6P/erA05GAOXijs/pki/BVcGsHNNjr1xRE0qU="
     # Doctr deploy key for bluesky/bluesky.github.io
     - secure: "aAdAJ0KVEaeEgqP+dMhkfjtTsORNnyKAwgI4TTyEiNw2VxofMc35jlArKdAW4ghMqaVv3+PrhLUEo27a36kWdSmmsXmbB4HUI5ef0qeiP3/OT+pvHFZ55rKnlKCn1Bcd2uTekkaH4ZPQSPB3mKdoM9TJbIchq+jfrHRAsXkNdCFubx3SDOS3n4Xp3iqF7udrsp2yPz2FuAxF4NdxoZIx/GeKAXTNNeRtzvvsY4hKWEk3ZaYF29gmszVefT4eLJAMd1UjaRJV/i+/cbXMtp73xpaF4/XzpwAWGcANcQxm+NVSNdTGPzsLA7t2FgfbTUWhdzf9OiHMGvK7Z0kjjiD2UOIQ9Tn26kwLYql5zvB69YLnvbAUJMp46BHEdDSGnUgWCYrxeGt+phU1UTjEitl5vEZxgWwYWx6w5B6zMAshXA3IE/WWVwNn3t3LNoQ/c5yP3u4g0d+XTMKrBi3dH3qqGN3Ukx4y6LY6bV3LpMUgtgLXzuEh6y67nSS4TUltc89Xx23zDfA4YrQJg+agJrNaBIUpqm/acJr8rQ1SsrTmY5nqFIUCCN2xrqgeafX+EmeX3xg0CGI/A7oFty8sLpPqY4SXoBIVg+8mvXBAmVV7ccqgwKcSo1x+M/HW7eloRL/GuDbhxwxn0l7LNlYTChYuFVE/A5x9OyGBHKfVGsjH2G8="
+    - MPLBACKEND: Qt5Agg
 
 cache:
   directories:
     - $HOME/.cache/pip
     - $HOME/.cache/matplotlib
 
+addons:
+  apt:
+    packages:
+      - qtbase5-dev
+
 matrix:
   fast_finish: true
   include:
-    - python: 3.6
+    - os: linux
+      python: 3.6
       env: PUBLISH_DOCS=1
-    - python: nightly
+    - os: linux
+      python: 3.7
+      sudo: true
+    - os: osx
+      language: generic
+      env: PYTHON=3.6
+    - os: osx
+      language: generic
+      env: PYTHON=3.7
+    - os: linux
+      python: nightly
   allow_failures:
     - python: nightly
 
 before_install:
-  - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16"
-  - "export DISPLAY=:99.0"
+  - |
+    if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+        pushd ${HOME}
+        wget https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh && \
```

Comment:
I don't think it's necessary to chain the lines with ` && \` here. You might also consider removing some of the debug lines below.

Reply:
Done. I left some `conda info/list` commands to have some information in which env we are running the tests.